src/Grid/grid.jl

```diff
@@ -392,8 +397,13 @@ Returns an iterateable face skeleton. The skeleton consists of `FaceIndex` that
 """
 faceskeleton(top::ExclusiveTopology, grid::AbstractGrid) =  top.face_skeleton
 
-toglobal(grid::Grid,vertexidx::VertexIndex) = vertices(getcells(grid,vertexidx[1]))[vertexidx[2]]
-toglobal(grid::Grid,vertexidx::Vector{VertexIndex}) = unique(toglobal.((grid,),vertexidx))
+"""
+    toglobal(grid::AbstractGrid, vertexidx::VertexIndex)
```

Comment:
```suggestion
    toglobal(grid::AbstractGrid, vertexidx::VertexIndex) -> Int
```

to make it consistent with the other comment?